osmo_jupyter/timezone.py

```diff
@@ -20,5 +20,5 @@ def utc_series_to_local(pandas_timeseries):
     '''
     definitely_a_timeseries = pd.to_datetime(pandas_timeseries)
```

Comment:
Fun but maybe something more definite...err...why is it definitely_a_timeseries?

Reply:
because the input may have been a series of strings. I had been meaning to clarify that.